Firmware/Marlin_main.cpp

```diff
@@ -1361,6 +1411,7 @@ void loop()
   manage_heater();
   isPrintPaused ? manage_inactivity(true) : manage_inactivity(false);
   checkHitEndstops();
+  host_keepalive();
```

Comment:
That sounds likely, I haven't been able to test it though. (See explaination below)